api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -299,4 +299,13 @@ message CommonTlsContext {
   // Custom TLS handshaker. If empty, defaults to native TLS handshaking
   // behavior.
   config.core.v3.TypedExtensionConfig custom_handshaker = 13;
+
+  // The path to save the TLS key log. The default value is /var/log/envoy/tlskey.log
```

Comment:
I don't think we should have a default here.  I think this field should be required, and if it's not specified, no key logging will be done.

Reply:
ok. I will change it.